clip.bash

```diff
@@ -81,9 +83,14 @@ cmd_clip() {
 
     # Figure out if we use fzf or rofi
     local prompt='Copy password into clipboard for 45 seconds'
-    local fzf_cmd="fzf --print-query --prompt=\"$prompt\" | tail -n1"
+    local fzf_cmd="fzf --print-query --prompt=\"$prompt\""
     local rofi_cmd="rofi -dmenu -i -p \"$prompt\""
```

Comment:
For `rofi` you can add `--filter \"$term\"`

Reply:
Done, thanks!